bing_search.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# !/usr/bin/env ruby
+
+require 'rest-client'
+
+# A simple Http Request using Rest-Client Bing search for ruby
+
+class BingSearch
+  attr_accessor :url, :querry
```

Comment:
Why use `attr_accessor` for `:url` ? 

It means one can overwrite the url of a BingSearch object after initialization. It is by design and how is that useful? If you don't intend to overwrite `:url` after instantiation, you should use `attr_reader` for that attribute.